lib/server/controllers/tags.controller.js

```diff
@@ -43,6 +43,9 @@ const createTag = async (request, response) => {
     if (!value) {
         return;
     }
+    if (value.body.color?.length === 6) {
```

Comment:
The `colorDto` which is part of `CreateTagDto` already checks for a length of 6.
Do we need to check again here?

Reply:
Updated